apps/admin/backend/src/cvr_files.ts

```diff
@@ -344,7 +348,9 @@ export function getAddCastVoteRecordReportErrorMessage(
   const errorType = error.type;
   switch (errorType) {
     case 'report-access-failure':
-      return 'Failed to access cast vote record report for import.';
+      return 'Unable to access cast vote record report for import.';
+    case 'cast-vote-records-authentication-error':
+      return 'Unable to authenticate cast vote records. Try re-exporting from the scanner.';
```

Comment:
Suggestion for slightly plainer language in the copy:

"Try exporting them from the scanner again."

Reply:
Good call - adjusted copy in a few places accordingly: https://github.com/votingworks/vxsuite/pull/3534/commits/6e8e5f57aa5879c0dc0dcee962966fe1262c14ca